dagshub/streaming/filesystem.py

```
@@ -123,12 +123,15 @@ def __init__(
-                    "or specify `repo_url` (url of repo on DagsHub) and "
-                    "`project_root` (path to the folder where to mount the filesystem) arguments"
-                )
+                if repo_url:
+                    self.project_root = Path(os.path.abspath("."))
+                else:
+                    raise ValueError(
+                        "Could not find a git repo. Either run the function inside of a git repo, "
+                        "specify `project_root` with the path to a cloned DagsHub repository, "
+                        "or specify `repo_url` (url of repo on DagsHub) and "
+                        "`project_root` (path to the folder where to mount the filesystem) arguments"
+                    )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)